daemon/cluster/executor/container/health_test.go

```diff
@@ -0,0 +1,102 @@
+// +build !windows
```

Comment:
> Also I notice that both docker_cli_swarm_test and docker_cli_service_update_test are built !windows

This is because they use second daemon binary. There maybe something here as well that doesn't work on windows and what I'm missing.